modules/about-microshift.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// microshift/understanding-microshift.adoc 
+
+[id="con-about-microshift_{context}"]
+= About MicroShift
+
+Working with low-resource field environments and hardware presents many challenges not experienced with cloud computing. {product-title} enables you to solve problems for edge devices deployed in the field by:
+
+* Overcoming the operational challenge of minimal system resources, for example, a {op-system-chip} with {op-system-ram}.
+* Addressing the environmental challenges of severe networking constraints such as low or no connectivity.
+* Meeting the physical constraint of hard-to-access locations by installing your system images directly on edge devices.
+* Building on and integrating with edge-optimized operating systems such as {op-system-first}.
+
+{product-title} has only those functions and services you need for computing in resource-constrained locations, but uses some {OCP} commands. With {product-title}, you have the simplicity of single-node deployment. You can have many deployments on different hosts, creating the specific system image needed for each of your applications. Installing {product-title} on top of your managed {op-system} devices in hard-to-service locations also allows for streamlined over-the-air updates.
```

Comment:
```suggestion
{product-title} has the simplicity of single-node deployment with the functions and services you need for computing in resource-constrained locations. You can have many deployments on different hosts, creating the specific system image needed for each of your applications. Installing {product-title} on top of your managed {op-system} devices in hard-to-service locations also allows for streamlined over-the-air updates.
```
Minimalism suggestion here. Since you present MicroShift as "a distribution of OCP", I don't see the need to state that MicroShift uses some OCP commands.

Reply:
Agreed.